app/index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>Wuf</title>
     <meta name="description" content="">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
```

Comment:
What does this do? I haven't used it before.

Reply:
This is part of what allows the app to respect the safeareas of the notch in an iPhone X or 11.